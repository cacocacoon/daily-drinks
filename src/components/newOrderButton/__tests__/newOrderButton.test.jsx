import React from 'react'
import TestRenderer from 'react-test-renderer';
import { MemoryRouter } from 'react-router-dom'

import NewOrderButton from '../newOrderButton'

describe('NewOrderButton', () => {
    it('default view', () => {
        const button = TestRenderer.create(
            <MemoryRouter>
                <NewOrderButton />
            </MemoryRouter>
        )
        expect(button.toJSON()).toMatchSnapshot()
    })
})