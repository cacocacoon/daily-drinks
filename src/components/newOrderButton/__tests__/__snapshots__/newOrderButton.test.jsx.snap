// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NewOrderButton default view 1`] = `
<a
  href="/order"
  onClick={[Function]}
>
  <button
    className="new-order-btn btn"
  >
    + Add
  </button>
</a>
`;
